provision/acc_provision/cloud_provision.py

```diff
@@ -169,12 +169,13 @@ def Run(self, flavor_opts, kube_yaml_gen_func):
         output_tar = self.args.output_tar
         operator_cr_output_file = self.args.aci_operator_cr
         gen(self.config, output_file, output_tar, operator_cr_output_file)
-        if self.args.flavor == "aks":
-            self.print_aks_setup()
-        elif self.args.flavor == "eks":
-            self.print_eks_setup()
-        else:
-            self.print_openshift_setup()
+        if not self.args.test_run:
```

Comment:
You will need to first check if test_run is present in args.

Reply:
Cool, makes sense I guess -> if not (self.args.test_run in self.args): OR if self.args.test_run not in self.args:

Reply:
this comment is still not addressed...